unit_tests/UnitTestGetDofStatus.C

```diff
@@ -0,0 +1,47 @@
+#include <gtest/gtest.h>
+
+#include <UnitTestHelperObjects.h>
+#include <stk_mesh/base/BulkData.hpp>
+
+#include "UnitTestUtils.h"
+
+#include <TpetraLinearSystem.h>
+
+TEST_F(Hex8MeshWithNSOFields, getDofStatus_basic)
```

Comment:
Any specific reason to use this with `Hex8MeshWithNSOFields` instead of a custom class? I would prefer it to be a separate class that can be used for filtering with `--gtest-filter`

Reply:
That's good feedback, I'll make it a separate class.